code-of-conduct.md

```diff
@@ -1,113 +1,83 @@
+To help us respond to your report, please include as much of the following information as possible in your reports:
+- The date and location of the incident
+- A description of the events.
+- The name or identity of the individuals involved in the incident.
+- The names or identities of any other witness to the incident.
+- Any documentation such as emails, private messages, photographs, police reports, texts or notes related to the incident.
+- Information for your preferred method of contact from our Code of Conduct Committee if you wish to or are willing to be contacted for follow up questions.
+
+## Code of Conduct resposne
+Code of Conduct response
+The Code of Conduct Committee will review all good faith reports and provide a response to the party making the report within two weeks, barring any legal requirement that prevents it.
+US-RSE and the Code of Conduct Committee will endeavor to resolve all Code of Conduct Reports within 60 days of receipt of complaint whenever possible.
+
+We will take all good faith reports of code of conduct violations by US-RSE members seriously.
+When responding to reports, we may take into account the behavior of members outside formal US-RSE spaces if in our judgment they are relevant to the US-RSE community.
+US-RSE reserves the right to deny or revoke membership to individuals based on past behavior, including conduct towards people who are not members of US-RSE.
+To preserve the time and resources of the Committee, incidents occurring outside US-RSE spaces should only be reported to the Committee if they have a serious impact on the safety of the US-RSE community.
+
+In order to preserve the integrity and health of the Code of Conduct Committee, the Code of Conduct Committee and US-RSE reserve the right to reject any report believed to have been made in bad faith or that is otherwise judged not relevant.
+Intentional false reports or reports that serve as a form of harassment/abuse or to silence legitimate criticism are themselves Code of Conduct violations.
+
+Members of the Code of Conduct Committee who have a genuine conflict of interest or otherwise cannot provide a fair assessment of your report will recuse themselves from handling your report.
+
+All reports to and deliberations of the Code of Conduct Committee will be kept confidential, and we will not publicly name those who report or experience suspected Code of Conduct violations, with the following limited exceptions.
+The US-RSE Code of Conduct and leadership reserves the right to publicize or inform third parties about an incident, including openly naming the offending party, if we believe that doing so is necessary to preserve the reputation and integrity of US-RSE, will increase the safety of US-RSE members, is required by a valid court order or other statutory requirement, or will increase the safety of the general public.
```

Comment:
"US-RSE Code of Conduct and leadership " - something missing here - maybe Code of Conduct committee?